Update Managing Content guide (SCA, Activation keys) (#1999)
theforeman/foreman-documentation: guides/common/modules/proc_creating-an-activation-key.adoc

You can use activation keys to define a specific set of subscriptions to attach to hosts during registration.
The subscriptions that you add to an activation key must be available within the associated Content View.

[IMPORTANT]
====
If you have Simple Content Access (SCA) enabled on {Project}, you cannot attach subscriptions to your activation key.

> With SCA enabled, subscriptions are automatically attached to your hosts.

Reviewer: keeping it technically accurate here :)
```suggestion
With SCA enabled, having subscriptions attached to your hosts is not needed.
```
